src/NServiceBus.AcceptanceTests/ConfigureEndpointLearningTransport.cs

```diff
@@ -39,8 +39,20 @@ public Task Configure(string endpointName, EndpointConfiguration configuration,
         //we want the tests to be exposed to concurrency
         configuration.LimitMessageProcessingConcurrencyTo(PushRuntimeSettings.Default.MaxConcurrency);
 
-        configuration.UseTransport<LearningTransport>()
-            .StorageDirectory(storageDir);
+        var transportConfig = configuration.UseTransport<AcceptanceTestingTransport>()
```

Comment:
Rename the file to `ConfigureEndpointAcceptanceTestingTransport.cs` ?

Reply:
this was just a demo of how you would use it, of course that would have to be named appropriately